domain-models-interface-extensions/pom.xml

```diff
@@ -11,6 +11,12 @@
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
+      <version>4.0.0.Beta3</version>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-client</artifactId>
+      <version>4.0.0.Beta3</version>
```

Comment:
Can you remove the version line so that is uses the version provided by the parent pom.xml?